.evergreen/config.yml

```diff
@@ -411,6 +411,17 @@ tasks:
   - func: "create packages and repos"
   - func: "upload packages and repos"
 
+- name: build-with-earthly
+  commands:
+  - func: "fetch source"
+  - func: "earthly"
+    vars:
+      args: +build --env=${earthly_env}
+  - func: "earthly"
+    vars:
+      args: --artifact +build/libmongocrypt-install ${workdir}/install
+  - func: "tar and upload libmongocrypt libraries"
+
```

Comment:
Suggest updating the `upload-all` task to include the Alpine tasks.

The tarball uploaded in the `upload-all` task appears [referenced in Python bindings](https://github.com/mongodb/libmongocrypt/tree/6b328b46d2489526b7cc013a9b3d36eddfe3ab62/bindings/python#installing-libmongocrypt), and [consumed in C# bindings](https://github.com/mongodb/libmongocrypt/blob/6b328b46d2489526b7cc013a9b3d36eddfe3ab62/bindings/cs/Scripts/build.cake#L23).

Aside: eventually I would like to remove/revise the `upload-all` task. The tarball is ~2GB, but I expect consumers only need a subset of builds. The task depends on slow-to-schedule distros.